examples/TeamMembers.ts

```diff
@@ -0,0 +1,18 @@
+import * as HelloSignSDK
+  from "@hellosign/openapi-javascript-sdk";
+
+const api = new HelloSignSDK.TeamApi();
+
+// Configure HTTP basic authorization: api_key
+api.username = "YOUR_API_KEY";
+
+// or, configure Bearer (JWT) authorization: oauth2
+// $config->setAccessToken("YOUR_ACCESS_TOKEN");
+
+const result = api.teamMembers("team_id");
```

Comment:
Same comment as other PR. For TypeScript, I believe there should be a specific type that gets used to pass `team_id` as part of a data object

Reply:
done